### PR TITLE
Careers - new page

### PR DESCRIPTION
**Pull request by galdranorn**

### CODE REVIEW

File: _includes/careers/hero.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  
+  const showVideo = () => {
+    clearInterval(timer);
+    $image.classList.add(hiddenClass);
+  };
+
+  const checkBuffer = () => {
+    if (event.target.getVideoLoadedFraction() > .5) {
+      showVideo();
+      return;
```
Comment (siaradona) [inline _includes/careers/hero.js:43]: `return showVideo();` (?)

File: _includes/careers/hero.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    'playsinline': 1,
+    'playlist': 'PZ9U8wR0F2c',
+    'rel': 0,
+    'version': 3,
+  }
+};
+
+let player;
+
+function onYouTubeIframeAPIReady() {
```
Comment (siaradona) [inline _includes/careers/hero.js:24]: arrow?

File: _includes/careers/path.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<div class="careers-path">
+  <div class="careers-path__image">
+    <img src="/assets/images/loader.gif" data-src="/assets/images/careers/path.webp" />
```
Comment (siaradona) [inline _includes/careers/path.html:3]: alt

File: _includes/slider.js [hunk 4]
```diff
@@ -15,11 +15,20 @@ window.addEventListener('load', () => {
...
   const sliderType = $slider[0].dataset.type;
+  let openCardClass;
+
+  if (sliderType === 'product-design') {
+    openCardClass = 'product-design__accordion-card--opened';
+  } else if (sliderType === 'careers') {
+    openCardClass = 'careers-organization__card--opened';
+  } else {
+    openCardClass = 'services-slider_card--opened';
+  }
```
Comment (siaradona) [inline _includes/slider.js:26]:
```
const incredibleName = {
  default: 'services-slider_card--opened'
  'product-desing': 'product-design__accordion-card--opened',
  'careers': 'careers-organization__card--opened',
  [...]
}

const openCardClass = incredibleName[sliderType]
```

File: _sass/new-styles/modules/careers/careers/_atmosphere.scss [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    &__content, &__image {
+      box-sizing: border-box;
+      width: 50%;
+    }
+
+    &__content-cta {
+      width: 300px;
+      
+      a {
+        background-image: linear-gradient(0deg, $color-secondary 0%, $color-secondary 50%, $color-home-grey 50%, $color-home-grey 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_atmosphere.scss:106]:
```
background-image: linear-gradient(0deg, $color-secondary 0%,
                                        $color-secondary 50%,
                                        $color-home-grey 50%,
                                        $color-home-grey 100%); 

```

File: _sass/new-styles/modules/careers/careers/_atmosphere.scss [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+@media(min-width: 992px) {
+  .careers-atmosphere {
+    &::before {
+      right: -100px;
+      width: 1050px;
+      border-radius: 8px;
+    }
+    
+
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_atmosphere.scss:136]: .

File: _sass/new-styles/modules/careers/careers/_cta.scss [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+  &__text {
+    font-size: 24px;
+    line-height: 40px;
+    color: #312E2B;
+
+    &--big {
+      display: block;
+      font-size: 40px;
+      line-height: 60px;
+      margin-bottom: 15px;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_cta.scss:23]: .

File: _sass/new-styles/modules/careers/careers/_hero.scss [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+  }
+
+  .btn-primary {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    width: 210px;
+    padding: 17px;
+    font-size: 13px;
+    background: linear-gradient(270deg, $color-secondary 0%, $color-secondary 50%, $color-white 0%, $color-white 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_hero.scss:53]: ..

File: _sass/new-styles/modules/careers/careers/_hero.scss [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+  .careers-hero {
+    &__content {
+      width: 40%;
+    }
+
+    &__image {
+      width: 60%;
+      margin-bottom: 5px;
+      height: auto;
+      padding-top: 0;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_hero.scss:165]: .

File: _sass/new-styles/modules/careers/careers/_hero.scss [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+
+    .geometric-figure {
+      &_center, &_right::after {
+        width: 260px;
+        height: 260px;
+      }
+  
+      &_right {
+        height: 260px;
+        width: 130px;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_hero.scss:202]: .

File: _sass/new-styles/modules/careers/careers/_involvement.scss [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+
+    .btn {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      width: 210px;
+      margin-top: 25px;
+      padding: 17px;
+      font-size: 13px;
+      background: linear-gradient(270deg, $color-secondary 0%, $color-secondary 50%, $color-white 0%, $color-white 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_involvement.scss:71]: ..

File: _sass/new-styles/modules/careers/careers/_organization.scss [hunk 12]
```diff
@@ -0,0 +1,414 @@
...
+      height: 490px;
+      width: 260px;
+      min-width: 260px;
+
+      &-content {
+        height: auto;
+        width: 260px;
+        min-width: 260px;
+        margin: 10px;
+        background: $color-white;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_organization.scss:200]: background-color

File: _sass/new-styles/modules/careers/careers/_path.scss [hunk 13]
```diff
@@ -0,0 +1,187 @@
...
+    background: $color-page-text;
+
+    .btn {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      width: 210px;
+      padding: 17px;
+      font-size: 13px;
+      background: linear-gradient(270deg, $color-secondary 0%, $color-secondary 50%, $color-white 0%, $color-white 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_path.scss:51]: ..

File: _sass/new-styles/modules/careers/careers/_recruitment.scss [hunk 14]
```diff
@@ -0,0 +1,179 @@
+.careers-recruitment {
+  &__header, &__steps {
+    box-sizing: border-box;
+    width: 95%;
+    max-width: 1200px;
+  }
+
+  &__header {
+    padding: 20px 20px 40px;
+    background: $color-geometric-blue-accent;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_recruitment.scss:10]: .

File: _sass/new-styles/modules/careers/careers/_recruitment.scss [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+      align-items: center;
+      justify-content: center;
+      position: absolute;
+      top: -18px;
+      left: 0;
+      width: 35px;
+      height: 35px;
+      color: $color-white;
+      border-radius: 100%;
+      background: $color-accent-yellow;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_recruitment.scss:69]: .

File: _sass/new-styles/modules/careers/careers/_recruitment.scss [hunk 16]
```diff
@@ -0,0 +1,179 @@
...
+
+      &--orange {
+        background: $color-secondary;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_recruitment.scss:72]: .

File: _sass/new-styles/modules/careers/careers/_recruitment.scss [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+        padding-top: 50px;
+        padding-right: 30px;
+      }
+
+      &-title {
+        margin-bottom: 20px;
+      }
+
+      &-text a {
+        background-image: linear-gradient(0deg, $color-secondary 0%, $color-secondary 50%, $color-white 50%, $color-white 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_recruitment.scss:135]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 18]
```diff
@@ -0,0 +1,239 @@
+.offers-blog {
+  padding: 50px 20px;
+  background: $color-page-text;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:3]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 19]
```diff
@@ -0,0 +1,239 @@
...
+      transition: border-radius .2s;
+
+      img {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: auto;
+        max-width: 130%;
+        height: 100%;
+
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:87]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 20]
```diff
@@ -0,0 +1,239 @@
...
+      }
+
+      &-image {
+        border-radius: 8px 8px 0 0;
+      }
+    }
+  }
+
+  .btn-bordered {
+    background: linear-gradient(270deg, $color-page-text 50%, $color-white 0%, $color-white 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:134]: ..

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 21]
```diff
@@ -0,0 +1,191 @@
...
+      flex-direction: column;
+      justify-content: space-between;
+      height: 250px;
+      width: 100%;
+      max-width: 310px;
+      margin: 20px auto 0;
+      padding: 30px;
+      border: 2px solid $color-secondary;
+      border-radius: 8px;
+      background: $color-white;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:53]: .

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 22]
```diff
@@ -0,0 +1,191 @@
...
+      bottom: 0;
+      z-index: -1;
+      width: 20px;
+      height: 20px;
+      border-radius: 100%;
+
+      &--yellow {
+        bottom: 650px;
+        right: 250px;
+        background: $color-accent-yellow;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:134]: .

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 23]
```diff
@@ -0,0 +1,191 @@
...
+      }
+
+      &--green {
+        bottom: 500px;
+        right: 0;
+        background: $color-estimate-dot-green;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:140]: .

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 24]
```diff
@@ -0,0 +1,191 @@
...
+      }
+
+      &--blue {
+        bottom: 100px;
+        right: 170px;
+        background: $color-geometric-blue;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:146]: .

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 25]
```diff
@@ -0,0 +1,191 @@
...
+      }
+    }
+  }
+}
+
+@media (min-width: 992px) {
+  .offers-openings__tiles {
+    &-item {
+      background: linear-gradient(270deg, $color-white 0%, $color-white 50%, $color-secondary 0%, $color-secondary 100%);
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:155]: ..

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 26]
```diff
@@ -0,0 +1,191 @@
...
+
+          .arrow-line {
+            fill: $color-white;
+          }
+        }
+      }
+    }
+
+    &-check {
+      transition: .2s;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:175]: `transition: all .2s` or `transition-duration: .2s`

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 27]
```diff
@@ -0,0 +1,191 @@
...
+  }
+}
+
+@media (min-width: 992px) {
+  .offers-openings__tiles {
+    &-item {
+      background: linear-gradient(270deg, $color-white 0%, $color-white 50%, $color-secondary 0%, $color-secondary 100%);
+      background-position: 100% 0%;
+      background-size: 200%;
+      transition: .2s;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:158]: .

File: _sass/new-styles/modules/careers/careers/_atmosphere.scss [hunk 28]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    &__content-cta {
+      width: 300px;
+      
+      a {
+        background-image: linear-gradient(0deg, $color-secondary 0%, $color-secondary 50%, $color-home-grey 50%, $color-home-grey 100%); 
+        background-size: 100% 200%;
+        background-position: 0 0;
+        transition: color .2s, background-position .2s;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_atmosphere.scss:109]: all (?)

File: _includes/careers/organization.html [hunk 29]
```diff
@@ -0,0 +1,82 @@
...
+              <a class="careers-organization__card-content__link" href="/career-offers">
+                Check our openings
+                {% svg /assets/images/btn-arrow.svg %}
+              </a>
+            </div>
+            <div class="careers-organization__card-hover">
+              <span class="slider-hoverBtn"><span class="icon icon-plus"></span></span>
+            </div>
+            <div class="careers-organization__card-background">
+              <img src="/assets/images/loader.gif" data-src="{{ perk.img }}" />
```
Comment (siaradona) [inline _includes/careers/organization.html:50]: alt

**Review by siaradona — CHANGES_REQUESTED** [review]

File: _includes/careers/hero.js (hunk 2 above)
Comment (galdranorn) [reply _includes/careers/hero.js:24]: this one function is not working this way

File: _sass/new-styles/modules/careers/careers/_atmosphere.scss (hunk 5 above)
Comment (galdranorn) [reply _sass/new-styles/modules/careers/careers/_atmosphere.scss:106]: + moved to _buttons.scss

File: _sass/new-styles/modules/careers/careers/_recruitment.scss (hunk 17 above)
Comment (galdranorn) [reply _sass/new-styles/modules/careers/careers/_recruitment.scss:135]: and moved to _buttons.scss

File: _includes/slider.js [hunk 30]
```diff
@@ -15,11 +15,17 @@ window.addEventListener('load', () => {
   const $sliderWrapper = $('[slider-wrapper]');
 
   const sliderType = $slider[0].dataset.type;
+  const openClasses = {
+    services: 'services-slider_card--opened',
+    careers: 'careers-organization__card--opened',
+    productDesign: 'product-design__accordion-card--opened' 
+  };
+  const openCardClass = openClasses[$slider[0].dataset.type];
```
Comment (siaradona) [inline _includes/slider.js:23]: `const openCardClass = openClasses[sliderType];`

File: _sass/buttons.scss [hunk 31]
```diff
@@ -203,6 +203,75 @@
...
+    box-shadow: none;
+  }
+}
+
+.btn-orange, .btn-orange-bordered {
+  width: 210px;
+  color: $color-white;
+  background-image: linear-gradient(270deg, $color-secondary 50%, 
+                                            $color-white 0%, 
+                                            $color-white 100%);
```
Comment (siaradona) [inline _sass/buttons.scss:235]: Why 50% is before 0%?

File: _sass/new-styles/modules/careers/careers/_involvement.scss [hunk 32]
```diff
@@ -0,0 +1,171 @@
...
+  .careers-involvement {
+    padding: 50px 80px;
+  }
+}
+
+@media(min-width: 811px) {
+  .careers-involvement {
+    display: flex;
+    flex-flow: row nowrap;
+    position: relative;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/careers/_involvement.scss:79]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 33]
```diff
@@ -0,0 +1,213 @@
...
+      border-radius: 8px;
+      overflow: hidden;
+    }
+
+    &-mask {
+      display: flex;
+      align-items: flex-end;
+      position: absolute;
+      top: 0;
+      left: 0;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:28]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 34]
```diff
@@ -0,0 +1,213 @@
...
+    line-height: 32px;
+    color: $color-white;
+  }
+
+  &__post {
+    margin-bottom: 30px;
+
+    &-wrapper {
+      display: block;
+      position: relative;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:17]: .

File: _sass/new-styles/modules/careers/offers/_blog.scss [hunk 35]
```diff
@@ -0,0 +1,213 @@
...
+      transform: translateY(-100%);
+      height: 100%;
+      width: 100%;
+      background-color: $color-white;
+      border-radius: 0 0 8px 8px;
+      transition: transform .4s;
+
+      .btn-white {
+        display: block;
+        position: relative;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_blog.scss:39]: .

File: _sass/new-styles/modules/careers/offers/_openings.scss [hunk 36]
```diff
@@ -0,0 +1,193 @@
...
+    &__header {
+      margin-bottom: 25px;
+      font-size: 40px;
+      line-height: 60px;
+    }
+
+    &__dot {
+      display: block;
+      position: absolute;
+      bottom: 0;
```
Comment (siaradona) [inline _sass/new-styles/modules/careers/offers/_openings.scss:125]: .

**Review by siaradona — CHANGES_REQUESTED** [review]

File: _sass/buttons.scss (hunk 31 above)
Comment (galdranorn) [reply _sass/buttons.scss:235]: to make "hard" not "smooth" transition between colors

**Review by siaradona — APPROVED** [review]

**Pull request merged**